rmgpy/data/kinetics/common.py

```diff
@@ -83,40 +83,44 @@ def sortEfficiencies(efficiencies0):
     if entry.label != '':
         f.write('    label = "{0}",\n'.format(entry.label))
 
+
+
     if isinstance(entry.item, Reaction):
-        if entry.label != str(entry.item):
```

Comment:
seems this `saveEntry` method is used for many purposes, can we also add some comments in the method to clarify which case (e.g., save to `groups.py`, `rules.py`, `reactions.py`) is for which if statement?